### DISCUSSION TITLE
How to rotate the RVE

### DISCUSSION THREAD

**Discussion by hnanhtuan** [neperfepx/neper#916] [Q&A] [answered]
Hi,

May I ask how can I rotate the RVE along an axis?
I am trying to use the following command to rotate the RVE but it does not work as expected.

`neper -T -loadtess n20-id2.tess -domain rotateaxis_x45 -o n20-id2_x45.tess
`

I verify the cell orientations before and after rotation, they are the same.
`
neper -T -loadtess n20-id2_x45.tess -for tess,ori -oridescriptor euler-bunge -statcell id
`
`
neper -T -loadtess n20-id2.tess -for tess,ori -oridescriptor euler-bunge -statcell id
`

Can someone please help me on this? 
Thank you very much.

Best regards,

Tuan Hoang

**Comment by rquey** [comment, marked as answer]
```
neper -T -loadtess n20-id2.tess -transform "rotate(1,0,0,45)" -o n20-id2_x45
neper -V n20-id2.tess -print n20-id2
neper -V n20-id2_x45.tess -print n20-id2_x45
```

![n20-id2](https://github.com/user-attachments/assets/bc6766ff-3831-44ae-95b9-683189c5ff2b)
![n20-id2_x45](https://github.com/user-attachments/assets/06a2e278-e11f-4b3c-ab8b-968b69c26948)
